.github/workflows/continuous-integration.yml

```diff
@@ -25,6 +25,7 @@ jobs:
           dotnet-version: |
             3.1.x
```

Comment:
I think you could (and should) remove outdated .NET versions which are no longer required